shell/monitor/monitor.sh

```diff
@@ -39,22 +39,34 @@ _monitor_set_experiment_info_func(){
   _monitor_experiment_info_func=( "$@" )
 }
 
+_monitor_get_code(){
+  local output=$1
+  echo "$output" | tail -n 1
```

Comment:
Or just `tail -n 1 <<< "$1"`

Reply:
Right, I was under the impression that tail <<< "$var" wouldn't work because "$var" wouldn't be terminated by a newline, but I just read in the man page that <<< *does* add a newline to its argument, so it will work fine.